File: Data Structures & Algorithms/is-anagram/submission-4.cpp
class Solution {
public:
    bool isAnagram(string s, string t) {
        if (s.length() != t.length()) {
            return false;
        }

        vector<int> alphabet(26, 0);
        for (int i = 0; i < s.length(); ++i) {
            alphabet[s[i] - 'a']++;
            alphabet[t[i] - 'a']--;
        }

        for (int& num : alphabet) {
            if (num != 0) return false;
        }

        return true;
    }
};

File: Data Structures & Algorithms/longest-consecutive-sequence/submission-4.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        seen = set(nums)
        res = 0

        for num in nums:
            if num - 1 in seen:
                continue
            curr = 1
            i = num + 1
            while i in seen:
                curr += 1
                i += 1
            res = max(curr, res)
        
        return res